canary/e2e/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "canary-cypress",
+  "version": "0.0.1",
+  "license": "MIT",
+  "private": true,
+  "main": "index.js",
+  "scripts": {
+    "clean": "rimraf node_modules",
+    "dev": "TZ=UTC cypress open",
+    "test": "TZ=UTC cypress run --spec 'features/**/*.feature'"
```

Comment:
OK! Let me try that.

Reply:
Actually, if we use the cypress in the e2e package, it will require to do a full install, for every package. This will significantly add to the run time, since it will need to install for every package. So let's just keep this separate cypress install for now.